docs/contributing/contributing_setup.md

```diff
@@ -154,6 +159,12 @@ Depending on which features of Great Expectations you want to work on, you may w
 	mysql_mysql_db_1   docker-entrypoint.sh mysqld   Up      0.0.0.0:3306->3306/tcp, 33060/tcp
 	````
 
+* Once the local mysql container is working, the tests against the mysql backend can be run using the `--mysql` flag. 
```

Comment:
When we aren't referencing it as a flag, we should use the official spelling of MySQL, same as you updated earlier from postgresql to PostgreSQL.
```suggestion
* Once the local MySQL container is working, the tests against the MySQL backend can be run using the `--mysql` flag. 
```

Reply:
I saw at least one place outside your edits where "mysql" is used instead of "MySQL", so I would do a search on the entire document and make certain you've updated all instances.